packages/create-plugin/templates/scenes-app/src/components/App/App.tsx

```diff
@@ -1,12 +1,50 @@
 import React from 'react';
+import { SceneApp, useSceneApp } from '@grafana/scenes';
 import { AppRootProps } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Alert } from '@grafana/ui';
+import { DATASOURCE_REF } from '../../constants';
 import { PluginPropsContext } from '../../utils/utils.plugin';
-import { Routes } from '../Routes/Routes';
+import { helloWorldPage } from '../../pages/HelloWorld/helloWorldPage';
+import { homePage } from '../../pages/Home/homePage';
+import { withDrilldownPage } from '../../pages/WithDrilldown/withDrilldownPage';
+import { withTabsPage } from '../../pages/WithTabs/withTabsPage';
```

Comment:
I don't have much knowledge when it comes to building Scenes apps but I believe this change means these pages are no longer code split. We added "route level" code splitting to the apps and scenes-apps templates to encourage developers to consider performance when starting out. Any chance we can continue to lazy load the pages by route?

Reply:
Sorry, I wasn't aware of these previous assumptions. Should I add back the Routes and figure out how to use them with the ScenesPages?

Reply:
Let's keep it as it is and take the question to the scenes team so we can get a better understanding of where / how to code split and get it documented. 👍